crates/astria-conductor/README.md

```diff
@@ -2,11 +2,11 @@
 
 Coordinates blocks between the data availability layer and the execution layer.
 
-### Running for development
+## Running for development
 
 * create `ConductorConfig.toml` in the repo root and populate accordingly, e.g.
 
-```
+```bash
```

Comment:
There are a few instances of ```` ```bash ````, but we mostly use ```` ```sh ````

Do we want to make sure we're consistent with these? It doesn't look like there is a rule to enforce this specifically.

Reply:
yeah we should use ````sh`